bin/plugin-server

```diff
@@ -37,7 +37,7 @@ if [[ -n $DYNO_RAM ]]; then
   export WORKER_CONCURRENCY=$(( ($DYNO_RAM - 1) / 512 + 1 ))
 fi
 
-cd plugins
+cd plugin-server
```

Comment:
👀

Reply:
This change broke plugin server

Reply:
oh wow, that wasn't there when I last reviewed it... 👀

Reply:
No way! The test don't run locally for me without that change 😱